docs/content/preview/deploy/manual-deployment/system-config.md

```diff
@@ -35,6 +38,33 @@ As of CentOS 8, `ntp` is no longer available and has been replaced by `chrony`.
 $ sudo yum install -y chrony
 ```
 
+### PHC (PTP Hardware Clock) {{<tags/feature/tp>}}
+
+Hardware clocks are more accurate.
```

Comment:
PTP Hardware Clock (PHC) refers to a physical clock device that supports the [Precision Time Protocol (PTP)](https://en.wikipedia.org/wiki/Precision_Time_Protocol) which is used for highly accurate time synchronization in computer networks. These hardware clocks are more accurate.

Reply:
Done